duckdb/spicepod.yaml

```diff
@@ -0,0 +1,12 @@
+version: v1beta1
+kind: Spicepod
+name: duckdb_quickstart
+datasets:
+  - from: duckdb:[database].[schema].[table]
+    name: [local_table_name]
+    params:
+      open: [path/to/duckdb/file/database_file.db]
+    acceleration:
```

Comment:
Lets leave out acceleration.  Instead you can make a step 5 "experiment with acceleration using Spice's data accelerators"